presto-lark-sheets/src/test/resources/testing-app-secret-base64

```diff
@@ -0,0 +1 @@
+ewogICJhcHAtc2VjcmV0IiA6ICJYSHdrckVna0N6Y3dFZ0NDNlpQQjRnWXZtb25OSmdNTiIKfQ==
```

Comment:
Is there a TTL for this secret? And is there an owner email account? There's a potential risk here that when the secret expires, it may break presto CI workflows.

Reply:
There is no TTL for the secret. The secret is not bound to an email account. 
I wonder if we should disable `TestSheetsIntegration` by default as `TestGoogleSheets`.

Reply:
Yeah, I think we may need to disable it.
Hi @beinan, could you help disable this integration test by default?

Reply:
Yes, let's disable the test, otherwise it would be too flaky I guess

We could use ```@Test(enabled = false)```  in the integration test.

Reply:
Disabled all the tests in `TestLarkSheetsIntegration`.